pyproject.toml

```diff
@@ -18,9 +18,15 @@ opentelemetry-instrumentation-flask = "^0.45b0"
 opentelemetry-instrumentation-requests = "^0.45b0"
 opentelemetry-exporter-otlp-proto-http = "^1.24.0"
 
-# for tests
-pytest = {version = "^8.2.0", optional = true}
-pytest-cov = {version = "^5.0.0", optional = true}
+requests = "^2.32.0"
+requests-kerberos = "^0.14.0"
+
+urllib3 = "^2.2.1"
+attrs = "^23.2.0"
+
+[tool.poetry.group.dev.dependencies]
+requests-mock = "^1.12.1"
+black = "^24.4.0"
```

Comment:
`black` should not be need since we already use `ruff` to auto-format the code in `pre-commit`.

Reply:
removed.